### DISCUSSION TITLE
Rock 4 C+

### DISCUSSION THREAD

**Discussion by HenriApperloo** [davesteele/comitup#223] [Q&A]
How can I use comitup on a Raspberry pi 4 clone, I test with Rock 4 C+ and ubuntu server 20.04.6 LTS

**Comment by davesteele** [comment]
Take a look at the [Installation Page](https://github.com/davesteele/comitup/wiki/Installing-Comitup) for some idea of the services that can conflict.

Ubuntu uses NetworkManager, which is required. That's a good first step.
